src/docs/howto-users-and-authentication/setup-oidc-proxy/index.md

```diff
@@ -473,3 +452,39 @@ their authentication flow using Passport.js or some other method and use
 the utility functions in `api-util/idToken.js` accordingly to wrap the
 login information into an OpenID Connect ID token that can be used to
 log in to a Sharetribe marketplace.
+
+## Troubleshooting your integration
+
+If your integration is not working as you would expect, we recommend
+that you check these things:
+
+### Inspect the id token
+
+You can log the `idpToken` created with the createIdToken function, and
+then paste it to [JWT.io](https://jwt.io/) to investigate what data is
+being included in the token. Make sure that any claims you are sending
+correspond to the
+[Open ID Connect standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).
+
+If there are claims missing, make sure that you are passing valid
+parameters
+[when creating the token in idToken.js](https://github.com/sharetribe/web-template/blob/main/server/api-util/idToken.js#L58-L63).
+
+### Make sure your .well-known endpoints are available in the correct address
+
+Check that your deployed application exposes
+**[your-marketplace-url]/.well-known/openid-configuration** and
+**[your-marketplace-url]/.well-known/jwks.json** endpoints, and that
+**[your-marketplace-url]** exactly matches the IdP URL defined for your
+identity provider client in Console
+
+### Double check your RSA key formatting
+
+Your RSA keys need to be correctly formatted in your deployment:
+
+- Include the `-----BEGIN ... KEY-----` and `-----END ... KEY-----`
+  lines in your environment variables
+- If you are passing a secret file (such as with Render), format the key
+  into a single string and replace line breaks with `\n`
+- If you are passing a multi-line environment variable (such as with
+  Heroku), copy the key with line breaks as-is.
```

Comment:
Possible addition:
- If you are running the template via ngrok and serving the RSA keys through environment variables, you might encounter an issue where the raw \n character is not treated as a newline character. This can lead to errors when trying to read the private or public key. To fix this, you can replace the raw \n character with the actual newline character. You can achieve this by using ```key.split(String.raw`\n`).join('\n')```